test/manifests/local-path-storage.yaml

```diff
@@ -0,0 +1,122 @@
+# create a kubernetes-in-docker cluster and replace the standard storage class with the local-path-provisioner.
+# fsGroups are not supported in the standard storage class, so we use the rancher local-path-provisioner.
```

Comment:
in `kubeaddons-enterprise` we end up using KIND `0.8.1` which comes up with local-path-provisioner already